management/sql/client.go

```diff
@@ -16,9 +17,14 @@ const (
 	azureCreateDatabaseURL = "services/sqlservers/servers/%s/databases"
 	azureGetDatabaseURL    = "/services/sqlservers/servers/%s/databases/%s"
 	azureListDatabasesURL  = "/services/sqlservers/servers/%s/databases?contentview=generic"
+	azureUpdateDatabaseURL = "/services/sqlservers/servers/%s/databases/%s"
 	azureDeleteDatabaseURL = "/services/sqlservers/servers/%s/databases/%s"
 
 	errParamNotSpecified = "Parameter %s was not specified."
+
+	DatabaseStateCreating = "Creating"
+
+	OperationPollInterval = 1 * time.Second
```

Comment:
1s is way too frequent. Our API throttles requests (I believe 1000 per subscription per hour). So consider making it 30s (if available/accessible, use the default in the ../management pkg)